cmd/pool/main.go

```diff
@@ -0,0 +1,63 @@
+package main
```

Comment:
I'm not following. Two commands? What's the reason for another command?

Reply:
Not going to persist, this was there for me to test with. I'll be removing with the final instance later.